config/ocp-monitoring/user-workload-monitoring-config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
This file belongs in the hack dir as it is a helper for OCP only. Loki benchmark should be unaware of any k8s flavour.